Sources/Paywalls/Components/PaywallImageComponent.swift

```diff
@@ -54,7 +54,6 @@ public extension PaywallComponent {
         public let overrideSourceLid: LocalizationKey?
         public let maskShape: MaskShape?
         public let gradientColors: [ColorHex]?
-        public let maxHeight: CGFloat?
```

Comment:
this is public, is it okay?

Reply:
Yup yup! Its behind the compiler flag still so nobody using 😇